content/blog/2025-11-10-tmil-22/index.md

```diff
@@ -0,0 +1,131 @@
+
+</figcaption>
+</figure>
+
+We released version [0.0.4 of the sparse strips][sparse strips 0.0.4] versions of the renderer.
+It should be a solid version of [Vello CPU](https://crates.io/crates/vello_cpu), suitable for a wider range of applications where CPU-only rendering is appropriate.
+In addition, it is the initial release of vello_hybrid, a newer approach that uses the GPU to do pixel compositing, but SIMD-accelerated geometry processing on the CPU.
+
+Vello CPU now supports non-isolated blending ([vello#1159][] among others), which is important for HTML5 canvas compatibility.
+SVG and COLRv1 emoji, by contrast, which has set priorities for Vello in the past, only support isolated blending.
+There are also promising early results for supporting image filters including blurs.
+
+A major development in Vello Hybrid is support for multiple image atlases ([vello#1252][]), which improves performance for image rendering.
+
+Thomas Smith has been making excellent progress on rendering sparse strip alpha values in GPU compute shaders.
+This was the original motivation for the sparse strip work.
+You can follow the progress in [#vello >  Thoughts on GPU sparse strips].
+
+Work on Vello Classic focused on support for web standard compatibility, motivated by Servo integration.
+Those features include support for non-premultiplied alpha ([vello#1173][], [vello#1262][], [vello#1145][]).
+This has been released in Vello v0.6.0, which also upgraded to wgpu v26 for compatibility with Bevy v0.17.
+
+
+## Masonry and Xilem
+
+Masonry is the widget system developed by Linebender.
+It provides a non-opinionated retained widget tree, designed as a base layer for high-level GUI frameworks.
+
+Xilem is our flagship GUI project, inspired by SwiftUI, which uses Masonry for its widgets.
+It lets you build user interfaces declaratively by composing lightweight views together, and will diff them to provide minimal updates to a retained layer.
+
+We released [version 0.4][Xilem 0.4.0] of both Masonry and Xilem in October.
+This release comprises 7 crates, including Xilem Web.
+It is the first release to switch over to the new HarfRust library, developed by Google Fonts, for shaping, replacing Swash.
+
+## Parley
+
+Parley is a text layout library.
+It handles text layout, mostly at the level of line breaking and resolving glyph positions.
+
+We released version [0.6 of Parley][parley 0.6.0].
+
+Slint has moved from their hand-rolled text implementation to Parley, and released version [1.14][Slint 1.14].
+Linebender is thrilled to have other UI toolkits in the ecosystem adopt our crates.
+
+Much of the development work on Parley is directed toward supporting HTML floats, particularly in Blitz.
+[parley#421][] is the main PR tracking this work, and references several other PRs.
+
+There has also been some performance work, which especially impacts larger paragraphs.
+This work continues, and we hope to publish quantitative benchmark results in November.
```

Comment:
What is this referring to? Is this the addition of Parley to Cosmic Text's benchmarks?